src/connectedk8s/azext_connectedk8s/custom.py

```diff
@@ -118,11 +124,15 @@ def create_connectedk8s(cmd, client, resource_group_name, cluster_name, https_pr
                                 summary="Couldn't find any node on the kubernetes cluster with the architecture type 'amd64' and OS 'linux'")
         logger.warning("Please ensure that this Kubernetes cluster have any nodes with OS 'linux' and architecture 'amd64', for scheduling the Arc-Agents onto and connecting to Azure. Learn more at {}".format("https://aka.ms/ArcK8sSupportedOSArchitecture"))
 
+    crb_permission = utils.can_create_clusterrolebindings(configuration)
+    if not crb_permission:
+        logger.error("Kubectl credentials doesn't have permission to create clusterrolebindings on this kubernetes cluster.")
```

Comment:
Better to terminate with failure here rather than waiting for the failure at helm install step. Also collect exception telemetry and set user fault